server/users.ts

```diff
@@ -642,7 +642,7 @@ export class User extends Chat.MessageContext {
 			return true;
 		}
 
-		let group: string = '';
+		let group: ServerGroup = ' ';
```

Comment:
This is a mechanical change and I'm not sure I like it.

Reply:
The initial value doesn't matter here, because it's assigned in an if-else so it's guaranteed to exist. Giving it a closer look though, not initializing it with any value (undefined) seems shorter.